README.rst

```diff
@@ -19,7 +19,7 @@ kernel symbols from Windows and Linux.
 
 If you would like higher level semantic information, then we suggest using 
 LibVMI with PyVMI (python wrapper, included with LibVMI) and Volatility.
-Volatility (http://code.google.com/p/volatility/) is a forensic memory analysis
+Volatility (https://github.com/volatilityfoundation/volatility) is a forensic memory analysis
```

Comment:
@tklengyel Rekall and Volatility seem pretty separate at this point, does it make more sense to mention them both or be more helpful to stick to Rekall since that what is used for its profiles?